src/vnmr/ds.c

```diff
@@ -167,7 +167,7 @@ static double  lvltlt_sense;
 static double  phasefine;
 static float  *cur_scl,scl_spec2;
 static float  *phase_data;
-static float  *curspec;
+static float  *curspec = NULL;
```

Comment:
just a comment: file-scope statics are always initialized to 0: http://stackoverflow.com/questions/13251083/the-initialization-of-static-variables-in-c